code/src/index.css

```diff
@@ -1,13 +1,77 @@
+@import url('https://fonts.googleapis.com/css2?family=Trispace:wght@400;700&display=swap');
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Trispace', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-size: 14px;
+  color: white;
+  background-image: url(./Images/background.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin: 0;
+  min-height: 1024px;
+}
+
+p {
+  font-weight: 500;
+  margin-top: 0px;
+}
+
+h1 {
+  font-size: 28px;
+  color: rgba(255, 255, 255, 0.973);
+  text-shadow: 2px 2px rgb(85, 94, 89);
+  text-align: center;
+  margin: 30px 0px 5px 0px;
```

Comment:
You don't need `px` if it is 0. So `margin: 30px 0 5px 0;` is enough.

Reply:
Thanks! Keep trying to remember that, I see that I do in some places, but not all 😄